src/components/NextVideoCard/NextVideoCard.jsx

```diff
@@ -1,31 +1,34 @@
+import { NavLink } from "react-router-dom";
 import "./NextVideoCard.scss";
 
 const NextVideoCard = (props) => {
-  const { video, handleClick } = props;
+  const { video } = props;
 
   const click = () => {
     handleClick(video.id);
   };
 
   return (
+    <NavLink className="navlink" to={`/videos/${video.id}` }>
```

Comment:
In dynamic url for react router it is important to start with "/followedByTheUrl"


to avoid concatenation of every path add to the url